docs/en/docs/features.md

```diff
@@ -24,10 +24,15 @@ Interactive API documentation and exploration web user interfaces. As the framew
 
 ![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-03-swagger-02.png)
 
-* Alternative API documentation with <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank"><strong>ReDoc</strong></a>.
+* Alternative API documentation with Redocly or Scalar
```

Comment:
"Redoc" instead of "Redocly" here as well.

Reply:
Nice :) Just pushed up a fix to this as well @nkamali